hera_mc/tests/test_weather.py

```diff
@@ -0,0 +1,171 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Testing for `hera_mc.weather`.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import nose.tools as nt
+import socket
+from math import floor
+import numpy as np
+from astropy.time import Time, TimeDelta
+from hera_mc import mc, weather
+from hera_mc.tests import TestHERAMC
+
+
+def is_at_katcp_enabled_site():
+    hostname = socket.gethostname()
+    if hostname == 'qmaster':
+        return True
+    else:
+        return False
```

Comment:
This could just be:

```python
return (socket.gethostname() == 'qmaster')
```

The parentheses are of course not syntactically necessary, but I find that they help clarify that we're evaluating a boolean expression.